src/components/Dropdown/Dropdown.tsx

```diff
@@ -339,20 +367,12 @@ const Dropdown = ({
     [multi, onClear, onSelect],
   );
 
-  // clickHandler will be used in onMouseDown to prevent the focus event
-  // opening the dropdown and the click closing it
-  const clickHandler = useCallback(
-    (e: React.MouseEvent) => {
-      e.preventDefault();
-      e.nativeEvent.stopImmediatePropagation();
-      setIsOpen(!isOpen);
-      if (containerInternalRef && containerInternalRef.current) {
-        // Focus the container even when clicking
-        containerInternalRef.current.focus();
-      }
-    },
-    [containerInternalRef, isOpen, setIsOpen],
-  );
+  const clickHandler = useCallback(() => {
+    if (containerInternalRef && containerInternalRef.current) {
+      // Focus the container even when clicking
+      containerInternalRef.current.focus();
+    }
+  }, [containerInternalRef]);
```

Comment:
IIRC refs changing don't trigger re-renders (but it's been a moment so @aVileBroker tell me if I'm wrong) so I _think_ L375 can be an empty array